psychopy/app/builder/dialogs/paramCtrls.py

```diff
@@ -39,6 +39,16 @@
 EVT_PARAM_CHANGED = wx.PyEventBinder(wx.IdManager.ReserveId())
 emptyNamespace = NameSpace(experiment.Experiment())
 
+# get if dark theme
+_appearanceSettings = wx.SystemSettings.GetAppearance()
```

Comment:
This seems to be breaking the test suite as it's querying wx before there's an app. I think it would make sense to move this code to a function (`getDarkMode`?) somewhere in utils, then import and call it when needed (e.g. in `styleValid` below)